main/src/vtr_gui/vtr_gui/vtr-gui/src/components/goal/GoalForm.js

```diff
@@ -115,8 +116,10 @@ class GoalForm extends React.Component {
                     variant="standard"
                     size="small"
                     error={goal_waypoints_invalid}
-                    onChange={(e) => this.setState({ goal_waypoints_str: e.target.value })}
-                    onKeyPress={this.parseGoalWaypoints.bind(this)}
```

Comment:
Is there a reason why the keyPress handler was removed?

Reply:
Mostly redundancy, since onChange pretty much always requires onKeyPress to have occurred (and we don't really care about the case when onKeyPress occurs but there's no change to the text box)
It's also a bit more direct since we intend parseGoalWaypoints() to be called whenever there's a change to the contents of the text box (so calling from onChange just feels more "correct")